grafana_wtf/model.py

```diff
@@ -168,11 +173,31 @@ def from_payload(cls, payload: any):
         if isinstance(payload, Munch):
             payload = dict(payload)
         if isinstance(payload, dict):
+            cls.validate(payload)
             return cls(**payload)
         if isinstance(payload, str):
             return cls(name=payload)
         raise TypeError(f"Unknown payload type for DatasourceItem: {type(payload)}")
 
+    @classmethod
+    def validate(cls, data: dict):
+        if "datasource" in data:
+            warnings.warn(
+                f"""
+The `datasource` attribute is ignored for the time being.
+
+See also: https://github.com/panodata/grafana-wtf/issues/110
+
+Please report back this occurrence to the grafana-wtf issue tracker,
+so the maintainers can improve the situation.
+
+The context of this error is in `DatasourceItem`, using this ingress data:
+{data}
+""".strip(),
+                UserWarning,
+            )
+            del data["datasource"]
```

Comment:
When the corresponding data payload includes the `datasource` attribute, tripping a runtime error at @apepojken's explorations, see GH-110, it will be ignored now. However, the user will be warned prominently about the situation, so we will hopefully get more feedback about it while we go.